api/HTMLVideoElement.json

```diff
@@ -513,7 +513,7 @@
               "version_added": false
             },
             "ie": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
I've confirmed https://mdn-bcd-collector.appspot.com/tests/api/HTMLVideoElement/msIsStereo3D returns false in IE 10 and 11.

And yet, https://docs.microsoft.com/en-us/previous-versions//hh772572(v=vs.85) says this was added in IE 10, but going by https://docs.microsoft.com/en-us/previous-versions/windows/apps/hh452749(v=win.10)?redirectedfrom=MSDN maybe it was only for some platforms, and there was a runtime flag for exposing the API.

I'm going to back out this change, because even if ultimately correct, it's a trickier case that deserves the visibility of its own PR, and maybe asking from friends from Microsoft.